content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md

```diff
@@ -19,6 +19,38 @@ You will need to have the following tools installed:
 - `cosign` ([install guide](https://docs.sigstore.dev/cosign/installation/))
 - `curl` (often provided by your operating system)
 
+## Verifying binary signatures
+
+The Kubernetes release process signs all binary artifacts (tarballs, SPDX files,
+standalone binaries) by using cosign's keyless signing. To verify a particular
+binary, retrieve it together with its signature and certificate:
+
+```bash
+URL=https://dl.k8s.io/release/v{{< skew currentVersion >}}.0/bin/linux/amd64
+BINARY=kubectl
+
+FILES=(
+    "$BINARY"
+    "$BINARY.sig"
+    "$BINARY.cert"
+)
+
+for FILE in "${FILES[@]}"; do
+    curl -sSfL --retry 3 --retry-delay 3 "$URL/$FILE" -o "$FILE"
+done
+```
+
+Then verify the blob by using `cosign`:
+
+```shell
+cosign verify-blob kubectl --signature kubectl.sig --certificate kubectl.cert
```

Comment:
Should we use `kubectl` or `$BINARY` here, e.g:

```suggestion
cosign verify-blob "$BINARY" --signature "$BINARY.sig" --certificate "$BINARY.cert"
```

Reply:
Good catch, fixed.